.github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md

```diff
@@ -0,0 +1,52 @@
+---
+name: 'Runtime Upgrade: [Upgrade Title/Version]'
+about: Checklist for tracking the progress of a runtime upgrade on the Paseo Testnet.
+title: ''
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation
+
+- [ ] **Upgrade Proposal Review**: Confirm that the upgrade proposal has been fully reviewed and approved.
```

Comment:
What do we exactly mean here? Is it the proposal of Polkadot? Some links might be worth

Reply:
i tried to write if the core team is aligned with the upgrade coming

Reply:
Let's make it more explicit 👍🏻

Reply:
already updated